src/components/addBookForm.js

```diff
@@ -1,42 +1,42 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 import React from 'react';
-// import uniqid from 'uniqid'
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/books/books';
 
 const AddBookForm = () => {
-  // const bookID = uniqid();
-  // const [bookTitle, setTitle] = useState(null);
-  // const [bookCategory, setCategory] = useState(null);
-
-  const updateTitle = () => { // the curly brace opens a multiline function
-    // setTitle(event.target.value)
-  };
-  const updateAuthor = () => { // the curly brace opens a multiline function
-    // setCategory(event.target.value)
-  };
-
-  // const newBook = {
-  //   id: bookID,
-  //   title: bookTitle,
-  //   category: bookCategory,
-  // };
-
-  const submitBook = () => {
+  const dispatch = useDispatch();
 
+  const submitBook = (e) => {
+    e.preventDefault();
+    const form = document.getElementById('form');
+    const newBook = {
+      id: uuidv4(),
+      categorie: document.getElementsByClassName('category').value,
+      title: document.getElementsByClassName('title')[0].value,
+      author: document.getElementsByClassName('author')[0].value,
+    };
```

Comment:
- [ ] Here you're taking the value according to the class element, but as required, is important to use the local state for react. In this case use hooks 'useState' to create the elements and then, assign it every time the value change (onChange inside the input), when the user clicks the button, take the values for the hooks and use it in place take it directly to the inputs. 🐶